python/tests/res/enkf/jobs/scaling/test_enkf_scaling_job.py

```diff
@@ -708,3 +708,23 @@ def test_main_entry_point_block_data_calc():
 
     for index, node in enumerate(obs_vector):
             assert node.getStdScaling(index) == 2.0
+
+
+@pytest.mark.usefixtures("setup_tmpdir")
+def test_validate_failed_realizations():
+    """
+    Config has several failed realisations
+    """
+
+    test_data_dir = os.path.join(_TEST_DATA_DIR, "local", "custom_kw")
+
+    shutil.copytree(test_data_dir, "test_data")
+    os.chdir(os.path.join("test_data"))
+
+    res_config = ResConfig("mini_fail_config")
+    ert = EnKFMain(res_config)
+
+    fs = ert.getEnkfFsManager().getCurrentFileSystem()
+    observations = ert.getObservations()
+
+    assert scaling_job.has_data(observations, ["GEN_PERLIN_1"], ert.getEnsembleSize(), ert.getEnkfFsManager().getCurrentFileSystem()) == []
```

Comment:
When GitHub activates the horizontal scrolling bar is a good hint that you should perhaps break your line ;)

Reply:
Yes, still cleaning that up a little bit, also realized that this allows an ensemble with no data to pass, which it should not, so adding a test for that and a fix.